etl/scripts-ccao-data-raw-us-east-1/housing/housing-ari_index.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import boto3
+import requests
+from botocore.exceptions import NoCredentialsError
+from bs4 import BeautifulSoup
+from dotenv import load_dotenv
+
+
+def get_most_recent_ihs_data_url(base_url):
+    headers = {
+        "User-Agent": (
+            "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+            "(KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3"
+        )
+    }
+    response = requests.get(base_url, headers=headers)
+    response.raise_for_status()
+
+    soup = BeautifulSoup(response.text, "html.parser")
+    links = soup.find_all("a", href=True)
+
+    xlsx_links = [
```

Comment:
This pulls an xlsx link rather than a specified sone, so if the string changes, the file will still work.